client/src/components/Header/components/nav-links.tsx

```diff
@@ -89,12 +95,24 @@ export class NavLinks extends Component<NavLinksProps, {}> {
     return previousSibling?.querySelector('a, button') ?? menuButtonRef.current;
   }
 
+  handleSignOutModal = (): void => {
+    console.log(showSignoutModal);
+    this.setState({ showSignoutModal: !showSignoutModal });
+  };
+
+  closeSignOutModal = (event: React.MouseEvent<HTMLButtonElement>): void => {
+    event.preventDefault();
+    // this is a nonsense but it's a nonsense that please typescript
+    const { isClose } = this.props;
+    isClose();
+  };
+
   handleLanguageChange = (event: React.MouseEvent<HTMLButtonElement>): void => {
     event.preventDefault();
     const { hideMenu, hideLanguageMenu, menuButtonRef, navigate } = this.props;
     const newLanguage = event.target.dataset.value as string;
     // If user selected cancel then close menu and put focus on button
-    if (newLanguage === 'exit-lang-menu') {
+    if (newLanguage === 'misc.sign-out') {
```

Comment:
A good question isn't why but how?

```suggestion
if (newLanguage === 'exit-lang-menu') {
```